src/content/docs/en/guides/internationalization.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Internationalization
 description: Learn how to use i18n routing in Astro.
-i18nReady: false
+i18nReady: true
```

Comment:
Oh no, how was _this_ page ever not translated?  :sweat_smile: 

This page will have updates on Thursday with the minor release though, so I'm going to say we should hold off marking this one just yet.
```suggestion
i18nReady: false
```